### ISSUE TITLE
QSlider markerLabels property (SliderMarkerLabels) can be Array<number|string> but not typed (TypeScript) nor documented

### ISSUE THREAD

**Issue by lugrinder** [quasarframework/quasar#17450]
### What happened?

If you use an array of numbers for markerLabels of QSlider works fine, but shows type error (vue-tsc) because SliderMarkerLabels has no definition to use Array<number|string>.

### What did you expect to happen?

It works, but I really don't know if this will be expected (not documented). As it works and is a valid and useful alternative (model is array index, label is array data), simply add to type (for TypeScript) and document.

### Reproduction URL

https://github.com/lugrinder/quasar

### How to reproduce?

Open project in an editor with Volar, or check using `vue-tsc`.

### Flavour

Quasar CLI with Vite (@quasar/cli | @quasar/app-vite)

### Areas

Components (quasar), TypeScript Support

### Quasar info output

```Shell
Operating System - Windows_NT(10.0.19045) - win32/x64
NodeJs - 20.13.1

Global packages
  NPM - 10.8.2
  yarn - Not installed
  @quasar/cli - 2.4.1
  @quasar/icongenie - 4.0.0
  cordova - Not installed

Important local packages
  quasar - 2.16.9 -- Build high-performance VueJS user interfaces (SPA, PWA, SSR, Mobile and Desktop) in record time        
  @quasar/app-vite - 1.9.5 -- Quasar Framework App CLI with Vite
  @quasar/extras - 1.16.12 -- Quasar Framework fonts, icons and animations
  eslint-plugin-quasar - Not installed
  vue - 3.4.38 -- The progressive JavaScript framework for building modern web UI.
  vue-router - 4.4.3
  pinia - 2.2.2 -- Intuitive, type safe and flexible Store for Vue
  vuex - Not installed
  vite - 2.9.18 -- Native-ESM powered web dev build tool
  eslint - 8.57.0 -- An AST-based pattern checker for JavaScript.
  electron - Not installed
  electron-packager - Not installed
  @electron/packager - Not installed
  electron-builder - Not installed
  register-service-worker - 1.7.2 -- Script for registering service worker, with hooks
  @capacitor/core - Not installed
  @capacitor/cli - Not installed
  @capacitor/android - Not installed
  @capacitor/ios - Not installed

Quasar App Extensions
  *None installed*
```

**Comment by rstoenescu** [comment]
@yusufkandemir can u take a look at this one, pls?

**Comment by yusufkandemir** [comment]
Thanks for reporting! The fix will be included in `quasar` v2.17.1
